phpunit.xml.dist

```diff
@@ -12,7 +12,6 @@
          stopOnIncomplete="false"
          stopOnSkipped="false"
          syntaxCheck="true"
-         strict="true"
```

Comment:
Why this was removed?

If you want to avoid deprecation notice from PHPUnit side, then better in separate PR swap this option with 4 options that allow to define fine-grained strict behavior.